### PR TITLE
feature(SessionStore): actions were moved.

### PR DESCRIPTION
**Pull request by PAKovalev**

### CODE REVIEW

File: packages/app/src/core/models/User/User.ts [hunk 1]
```diff
@@ -26,7 +26,7 @@ const User = types
   .views((self) => ({
     get avatarSrc(): string | undefined {
       return (
-        self.profile?.avatarHash &&
+        self.profile.avatarHash &&
```
Comment (aminbksi) [inline packages/app/src/core/models/User/User.ts:29]: I think you can drop it `self.profile.avatarHash &&`

**Review by aminbksi — APPROVED** [review]

File: packages/app/src/scenes/widgets/stores/ProfileMenuStore/ProfileMenuStore.ts [hunk 2]
```diff
@@ -13,13 +15,17 @@ const ProfileMenuStore = types.compose(
       profileMenuDialog: types.optional(Dialog, {}),
       tokenRulesDialog: types.optional(Dialog, {}),
       tokenRuleReviewStore: types.optional(TokenRuleReviewStore, {isWorldList: true}),
-      tokenRulesStore: types.optional(TokenRulesStore, {})
+      tokenRulesStore: types.optional(TokenRulesStore, {}),
+      statusRequest: types.optional(RequestModel, {})
     })
     .actions((self) => ({
-      openProfileMenu() {
+      openProfileMenu(): void {
```
Comment (Daboo1998) [inline packages/app/src/scenes/widgets/stores/ProfileMenuStore/ProfileMenuStore.ts:22]: Why? we never add void to actions and its not required.

**Review by Daboo1998 — APPROVED** [review]

**Review by dmitry-yudakov — APPROVED** [review]

File: packages/app/src/core/models/User/User.ts (hunk 1 above)
Comment (PAKovalev) [reply packages/app/src/core/models/User/User.ts:29]: what? it is without `?` )

Comment (aminbksi) [reply packages/app/src/core/models/User/User.ts:29]: what? you're returning `string`, avatar hash, also I think you can drop `undefined`

**Pull request merged**